src/circuit_json.rs

```diff
@@ -112,16 +135,19 @@ pub enum OpBox {
         id: BoxID,
         /// Number of qubits.
         n_qubits: u32,
-        qubit_indices: Vec<(u32, u32)>,
+        qubit_indices: Vec<(u32, u32)>, // TODO: come back to this one.
```

Comment:
Ah I flagged this as I thought this type was meant to be a list of pairs of registers (including name and multiple indexes) and integers, not a list of integers and integers

Reply:
Can you add that to the TODO?